src/BenchmarkDotNet.Diagnostics.Windows/PmcDiagnoser.cs

```diff
@@ -231,6 +234,9 @@ public MisspredictRateColumn(Dictionary<Benchmark, PmcStats> results)
             public bool AlwaysShow => false;
             public ColumnCategory Category => ColumnCategory.Diagnoser;
             public int PriorityInCategory => 1;
+            public QuantityType QuantityType => QuantityType.None;
+            public string GetName(ISummaryStyle style) => ColumnName;
```

Comment:
It should not, because inclusion of percentage sign doesn't change the magnitude of the value: `0.123 == 12.3%`
We worry about this in case of units of measure , because when we remove the unit from `0.123kB` or `123B`, we need to include it in the header for the value to be meaningful.

In other words, `GetValue` of `PmcDiagnoser` renders either `0.123` or `12.3%`, but never `12.3` which is the only case that would warrant `[%]` in the column header.

Do you think this warrants a comment in the code?